src/components/site/SummaryList.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const SummaryList = props => {
+  const { items = [] } = props;
+  return (
+    <React.Fragment>
+      {items.map(item => (
+        <div className="dg_summary-list">
+          <div className="dg_summary-list-items">
```

Comment:
this would be `dg_summary-list__item` since this is inside the loop.

Reply:
after running your example, i think you over-complicated this a bit. I think it would first help out to split the item to it's own component. Then check out this for a more simple way to accomplish this layout.

https://codepen.io/martypowell/pen/wvwZOrK

Reply:
done

Reply:
Looking at the Codepen, I agree this would  simplify the markup a bit, especially if we're using flexbox. My only feedback (after looking at the Codepen Marty attached) is would we want to wrap the <i> that is on the left, in a <div> as well?